src/_lib/sass/src/_labels.scss

```diff
@@ -34,8 +34,8 @@
         border-color: get-color("tertiary");
     }
     &.label-beta {
-        color: get-color("gray");
-        background: get-color("alt");
+        color: get-color("alt");
```

Comment:
what is alt here ?

Reply:
https://newswhip.github.io/style-guide/#/colors

Reply:
thanks